orcid-core/src/main/java/org/orcid/core/oauth/OrcidOauth2UserAuthentication.java

```diff
@@ -169,7 +169,7 @@ public String getName() {
             return StringUtils.EMPTY;
         }
         
-        if(Visibility.PUBLIC.value().equals(recordName.getVisibility())) {
```

Comment:
apologies if I got it wrong but if `.getVisibility()` returns a `Visibility` enum why couldn't it be `Visibility.PUBLIC`? 
A comparison on the enum value would indeed require `recordName.getVisibility().value().equals(Visibility.PUBLIC.values())` but that it isn't necessary to assess that the enum is `PUBLIC`?